ext/sigar/CMakeLists.txt

```diff
@@ -9,28 +9,29 @@ project(sigar)
 
 #--------------------------------------------------------------------
 # Define defintions
-if(WIN32) 
-    ## make sure we only use the smallest set of 
-    ## headers on win32. Otherwise we get clashes 
+if(WIN32)
+    ## make sure we only use the smallest set of
+    ## headers on win32. Otherwise we get clashes
     ## between winsock2.h and winsock.h
     add_definitions(-DWIN32_LEAN_AND_MEAN)
 endif()
 
 source_group("CMake Files" FILES ${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.txt)
 
-set(HEADER_FILES include/sigar.h 
-    include/sigar_fileinfo.h 
-    include/sigar_format.h 
-    include/sigar_getline.h 
-    include/sigar_log.h 
-    include/sigar_private.h 
-    include/sigar_ptql.h 
+set(HEADER_FILES include/sigar.h
+    include/sigar_fileinfo.h
+    include/sigar_format.h
+    include/sigar_getline.h
+    include/sigar_log.h
+    include/sigar_private.h
+    include/sigar_ptql.h
     include/sigar_util.h
 )
 
 ## linux
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     set(SIGAR_SRC src/os/linux/linux_sigar.c)
+    find_package(TIRPC REQUIRED)
```

Comment:
TIRPC, is that something that all Unix dist comes with or could this potentially introduce new CMake problems on some Unix system?